### ISSUE TITLE
[Meteor 2.5-beta.2] Error

### ISSUE THREAD

**Issue by nextmol** [meteor/blaze#354]
Hi,
Maybe Blaze is not compatible with Meteor 2.5? My application don't start when updating from 2.4 to any 2.5 beta version.

```
hot.js:19 Uncaught TypeError: Cannot read properties of undefined (reading 'id')
    at Blaze.Template.Template.onCreated (hot.js:19)
    at new BlazeRenderer (renderer.js:60)
    at new Router (router.js:15)
    at module (_init.js:33)
    at fileEvaluate (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:388)
    at Module.require (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:270)
    at require (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:310)
    at ostrio_flow-router-extra.js?hash=edcfb353524361cd223900b7346f46763f6a4b53:4484
    at ostrio_flow-router-extra.js?hash=edcfb353524361cd223900b7346f46763f6a4b53:4489
Template.onCreated @ hot.js:19
BlazeRenderer @ renderer.js:60
Router @ router.js:15
module @ _init.js:33
fileEvaluate @ modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:388
require @ modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:270
require @ modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:310
(anonymous) @ ostrio_flow-router-extra.js?hash=edcfb353524361cd223900b7346f46763f6a4b53:4484
(anonymous) @ ostrio_flow-router-extra.js?hash=edcfb353524361cd223900b7346f46763f6a4b53:4489
hot-module-replacement.js?hash=0f3ddf87865a399202cd7fe0b59b8f9601520795:330 HMR: connected
```
Thanks

**Comment by StorytellerCZ** [comment]
Confirmed. @zodern this looks like something with HMR. If you remove `hot-module-replacement` things work as usual.

**Comment by StorytellerCZ** [comment]
The latest 2.5 beta (beta.4) makes Blaze run. Just getting one error:
```
Uncaught TypeError: can't access property "id", currentModule is undefined
    onCreated hot.js:19
    module main.js:6
    module app.js:117
    fileEvaluate modules-runtime-hot.js:388
```

**Comment by filipenevola** [comment]
It should be fixed in Meteor 2.5-rc.1 https://github.com/meteor/meteor/pull/11645
